ckan_cloud_operator/providers/ckan/instance/cli.py

```diff
@@ -238,35 +247,45 @@ def solr():
 
 
 @solr.command('check')
-def solr_check():
+@click.argument('INSTANCE_ID')
+def solr_check(instance_id):
     '''
     Check the search index
     '''
+    manager.run_solr_commands(instance_id, 'check')
 
 @solr.command('clear')
-def solr_clear():
+@click.argument('INSTANCE_ID')
+def solr_clear(instance_id):
     '''
     Clear the search index
     '''
+    manager.run_solr_commands(instance_id, 'clear')
 
 @solr.command('rebuild')
-def solr_rebuild():
+@click.argument('INSTANCE_ID')
+def solr_rebuild(instance_id):
     '''
     Rebuild search index
     '''
+    manager.run_solr_commands(instance_id, 'rebuild')
 
 @solr.command('rebuild-fast')
-def solr_rebuild_fast():
+@click.argument('INSTANCE_ID')
+def solr_rebuild_fast(instance_id):
     '''
     Reindex with multiprocessing
     '''
+    manager.run_solr_commands(instance_id, 'check')
 
 @solr.command('show')
+@click.argument('INSTANCE_ID')
 @click.option('--dataset', help='Dataset name to show index for')
```

Comment:
Just for consistency I would call this parameter `dataset_id` since it is the name of it in the function call.

Reply:
:+1: